5001-6000/LIT5185PrayerHemamBuda.xml

```diff
@@ -52,9 +54,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <body>
             <div type="bibliography">
                 <listRelation>
-                    <relation name="lawd:hasAttestation" active="LIT5185PrayerHemamBuda" passive="ETH1310Buda"/>
+                    <relation name="saws:isRelatedTo" active="LIT5185PrayerHemamBuda" passive="ETH1310Buda"/>
```

Comment:
I think that ETH1310Buda refers to the Ethnic group (Fuga)

Reply:
ok, dropped.